workspace-names-updater@workspace-manager.gitlab.io/info.json

```diff
@@ -0,0 +1,3 @@
+{
+    "author": "Beatlink (https://gitlab.com/BeatLink)"
```

Comment:
This field just needs to contain the username, I think it's parsed by a script for the website.

Reply:
Done!